src/main/java/org/apache/maven/plugins/changes/ChangesReportGenerator.java

```diff
@@ -226,8 +224,8 @@ private void constructAction(Sink sink, ResourceBundle bundle, Action action) {
             sink.text(".");
         }
 
-        if (StringUtils.isNotEmpty(action.getDueTo()) || (!action.getDueTos().isEmpty())) {
-            constructDueTo(sink, action, bundle, action.getDueTos());
+        if (!action.getDueTos().isEmpty()) {
```

Comment:
I guess dueTos can't be null here?

Reply:
yes at least it will be an empty collection